typings/slickgrid.d.ts

```diff
@@ -1439,6 +1439,7 @@ declare namespace Slick {
 
 		// Single column returned
 		sortCol?: Column<T>;
+		//TODO: change this to 3 states?
```

Comment:
Maybe change the `TODO:` label to `TODO chsuh:`. It won't get lost among all the other "todo" comments if there's many. If there's only a few than it's probably fine to keep like this.